clients/cypress-e2e/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
```

Comment:
We won't have fixture for mocking requests. Maybe we'll find them useful for defining a bunch of test cases, but for now this might be misleading.

Reply:
good point, will delete!